include/silo/query_engine/actions/insertions.h

```diff
@@ -36,7 +36,8 @@ namespace silo::query_engine::actions {
 template <typename SymbolType>
 class InsertionAggregation : public Action {
    static constexpr std::string_view POSITION_FIELD_NAME = "position";
-   static constexpr std::string_view INSERTION_FIELD_NAME = "insertions";
+   static constexpr std::string_view INSERTED_SYMBOLS_FIELD_NAME = "inserted_symbols";
```

Comment:
```suggestion
   static constexpr std::string_view INSERTED_SYMBOLS_FIELD_NAME = "insertedSymbols";
```

I think in all other places we use camel case.